### PR TITLE
Fix issues on MacOS with ffmpeg

### PR DESCRIPTION
**Pull request by twollgam**
This PR fixes issues on MacOS with ffmpeg. 

Some time ago the homebrew package for ffmpeg@4 does not provide the av*-libs anymore for cmake projects. Building ffmpeg from source solves the problems.

### CODE REVIEW

**Review by ooshlablu — APPROVED** [review]
LGTM

File: .github/workflows/macos.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
         cmake
         cpprestsdk
         dylibbundler
-        ffmpeg@4
+        ffmpeg@6
```
Comment (Baklap4) [inline .github/workflows/macos.yml:36]: Can't we just remove this entirely as we're building from ffmpeg soure?

**Comment by Lord-Kamina** [comment]
Walk ne through what's going on here? This looks messy AF

File: .github/workflows/macos.yml (hunk 1 above)
Comment (twollgam) [reply .github/workflows/macos.yml:36]: ffmpeg@6 is pulled in by dependencies ...

But I think someone (@ooshlablu ;-) ) takes the changes and integrate the needed into his pr.

Comment (twollgam) [reply .github/workflows/macos.yml:36]: @Baklap4 so yes, we can remove the entry :)

File: .github/workflows/macos.yml [hunk 2]
```diff
@@ -78,6 +78,12 @@ jobs:
         id: install_deps
         run: |
            brew install $MACOS_DEPS
+
+      - name: Install FFMPEG Dependencies
```
Comment (twollgam) [inline .github/workflows/macos.yml:82]: The name is not matching. Better `Install dependency FFMPEG from source`?

**Review by ooshlablu — APPROVED** [review]

**Pull request merged**